DOC-2181: add fix documentation for TINY-10174 to the 6.8 release notes. (#3003)
tinymce/tinymce-docs: modules/ROOT/pages/6.8-release-notes.adoc

{productname} 6.8 addresses this issue, now the editor removes the redundant second initialization of the toolbar, and fix the 'fontsizeinput' 'destroy' action, so all its components are now properly unmounted. This fix ensures that updates are only applied to components that genuinely exist.

=== Dialog collection items would not display any icons chosen from icon pack.
// #TINY-10174

> In earlier versions of {productname}, the collections dialog component encountered an issue where items added to the collections failed to fetch the specified icon set in the "icon" property.
> 
> {productname} 6.8 addresses this issue, now, the collections dialog component correctly retrieves the icon specified in the "icon" property. As a fallback measure, if the specified icon cannot be retrieved, {productname} will use the string set within the property.

Reviewer: ```suggestion
In earlier versions of {productname}, the collections dialog component encountered an issue where items added to the collections would not display any icon as set in the `+icon+` property, instead only displaying the `+string+` itself.

In {productname} 6.8, dialog collection items will now use any specified `+icon+` from the xref:icons.adoc#icons[icon pack], with the `+string+` itself used in the case where no such icon exists. This is the current behaviour, as in most uses of dialog collections in {productname}, the `+icon+` property is set as an individual character, such as in the xref:emoticons.adoc[Emoticons] and xref:charmap.adoc[Charmap] plugins.
```
The previous behaviour is not now a fallback, but instead the intended behaviour in most cases. This is except for when an icon from the icon pack is specified, in which case we must confirm that first before using the individual character. This was the least complex/breaking solution to this problem.

{productname} 6.8 addresses this issue, now the editor removes the redundant second initialization of the toolbar, and fix the 'fontsizeinput' 'destroy' action, so all its components are now properly unmounted. This fix ensures that updates are only applied to components that genuinely exist.

=== Dialog collection items would not display any icons chosen from icon pack.
// #TINY-10174

> In earlier versions of {productname}, the collections dialog component encountered an issue where items added to the collections would not display any icon as set in the `+icon+` property, instead only displaying the `+string+` itself.

Reviewer: ```suggestion
In earlier versions of {productname}, collection items set in a xref:dialog-components.adoc#collection[collection dialog component] would not display any icon as set in the `+icon+` property, instead only displaying the `+string+` itself.
```
One last thing, green light once this is in.
